doc/content/3.documentation/1.concepts/4.exceptions.md

```diff
@@ -318,6 +318,12 @@ In any production system faults will happen and you should be prepared to manage
 
 You can use broker built-in tools like RabbitMQ Management UI, Azure portal Service Bus Explorer, Amazon AWS SQS web console, or Cogent Queue Explorer to inspect the Faults. Once the reason for the fault has been resolved, you can use the tool to extract the original message and send it back to the original consumer. In this manner, messages that fail Retries and Redelivery can still be successfully processed at a later stage.
 
+Also see:
+
+- [Retrying messages with the RabbitMQ Management UI](/documentation/transports/rabbitmq#retrying-messages)
+- [Retrying messages with the Azure portal Service Bus Explorer](/documentation/transports/azure-service-bus#retrying-messages)
+- [Retrying messages with the Amazon AWS SQS web console](/documentation/transports/amazon-sqs#retrying-messages)
```

Comment:
```suggestion
- [Retrying messages with RabbitMQ](/documentation/transports/rabbitmq#retrying-messages)
- [Retrying messages with Azure Service Bus](/documentation/transports/azure-service-bus#retrying-messages)
- [Retrying messages with Amazon SQS](/documentation/transports/amazon-sqs#retrying-messages)
```

I think it might be better to state the transport only, so that we can include more ways of doing retries in the details